services/ibge.js

```diff
@@ -1,6 +1,12 @@
 import axios from 'axios';
 
-const urlUf = ' https://servicodados.ibge.gov.br/api/v1/localidades/estados';
+const urlUf = 'https://servicodados.ibge.gov.br/api/v1/localidades/estados';
 
 export const getUfs = () => axios.get(urlUf);
 export const getUfByCode = (code) => axios.get(`${urlUf}/${code}`);
+
+export const getDistrictsByUf = async (uf) => {
+  const { data } = await axios.get(`${urlUf}/${uf}/distritos`);
```

Comment:
O data sempre vai existir? E se o request falhar?

Reply:
E se o data for null? Ou qualquer coisa, menos um array? Erro né...

Reply:
Alguma sugestão? O que fiz é que se falhar ele deve pegar os dados da wikipedia como fallback